uyuni-tools/reactivate_proxy_clients.py

```diff
@@ -0,0 +1,100 @@
+#!/bin/python3
+
+"""
+This script reactivates all clients which uses specified proxy FQDN as salt master.
+
+Workflow:
+
+1) use salt grains targeting to get the list of salt client ids
```

Comment:
It's a bit of a nitpick, but I think it brings clarity to change `client id` -> `minion id` since `client id` could be confused with the `system id` that we're using in Uyuni.
```suggestion
1) use salt grains targeting to get the list of salt minion ids
```